lib/patches/bundler/bundler/ruby_version.rb

```diff
@@ -15,7 +15,7 @@ def self.system
                             when "ruby"
                               ruby_version
                             when "rbx"
-                              Rubinius::VERSION.dup
+                              Truffle::Support::VERSION.dup
```

Comment:
This should not be modified (it's upstream code)

Reply:
Fixed.